frontend/projects/upgrade/src/dev-tools/key-value-bubble/key-value-bubble.component.ts

```diff
@@ -4,10 +4,11 @@ import { IKeyValueBubbleStyles } from './key-value-bubble.model';
 import { KeyValueBubbleService } from './key-value-bubble.service';
 
 @Component({
```

Comment:
@bcb37 , this change has introduced a lint problem and a similar problem in all the files. Can you check?

Reply:
omg we should just delete this whole dev-tools module, I forgot it existed.